doc/meetings/2023_04_07_db.md

```diff
@@ -0,0 +1,77 @@
+# Meeting 2023-04-07 DB
+
+_Protocolled by: Nursultan_
+
+## Attendees
+
+- Nursultan
+- Moritz
```

Comment:
```suggestion
```
I could not attend this meeting unfortunately.

Reply:
@badnames it will be today, it was moved.

Reply:
In one hour :wink: